### PR TITLE
Driver screen

### LINKED ISSUE #108: Add driver home screen
Should have the option to accept/reject a pickup assignment.

The flow:
1. Back office assigns a container 
1. (Driver receives a notification - to be done later)
1. Driver sees it on their screen along with other assigned containers
1. Accepts/ rejects the assignment
1. Once accepted, has the option to update status to Delivered when done.

### PR DESCRIPTION
**Pull request by anshul2401**
Closes #108

### CODE REVIEW

**Comment by anshul2401** [comment]
issue resolved

File: lib/screens/container/driver_list.dart [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                        : Colors.white),
+              ),
+              subtitle: Text('${t.origin} to ${t.destination}'),
+              children: [
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    textSpan('Assigned by: ', t.tenantId, c2: Colors.orange),
+                    textSpan('Size: ', t.size.toString(), c2: Colors.deepPurple)
```
Comment (harsimranmaan) [inline lib/screens/container/driver_list.dart:62]: The colors make it very hard to read. Can we stick with default colors until the color has some significance.

**Comment by harsimranmaan** [comment]
There is no indication on the UI to tell that some deliveries need an action.

File: lib/screens/container/driver_list.dart (hunk 1 above)
Comment (harsimranmaan) [inline lib/screens/container/driver_list.dart:62]: Use .text from the containersize extensions. toString() should be avoided on enums

**Review by harsimranmaan — APPROVED** [review]

**Pull request merged**